### PR TITLE
Install libllbuild swift

### PR DESCRIPTION
**Pull request by aciidb0mb3r**

### CODE REVIEW

**Comment by aciidb0mb3r** [comment]
Depends on https://github.com/apple/swift-llbuild/pull/224

File: products/llbuildSwift/CMakeLists.txt [hunk 1]
```diff
@@ -5,8 +5,38 @@ set(SOURCES
...
 list(APPEND additional_args -I ${CMAKE_CURRENT_SOURCE_DIR}/../libllbuild/include -lllbuild)
+list(APPEND additional_args -target x86_64-apple-macosx10.10)
 
 # Add swift bindings target if swift compiler is present.
 if (SWIFTC_FOUND)
   add_swift_module(libllbuildSwift llbuildSwift libllbuild "${SOURCES}" "${additional_args}")
+
+  # Install libllbuild.
+  install(DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR}/../libllbuild/include
+    DESTINATION lib/swift/pm/llbuild
```
Comment (ddunbar) [inline products/llbuildSwift/CMakeLists.txt]: This is ok, but in an ideal world would be a little more decoupled from Swift, and we would just expose the option and a build setting which defines the install location, and the Swift build script would configure it appropriately.

I don't feel strongly enough about it to advocate fixing it for this effort though.

Comment (aciidb0mb3r) [reply products/llbuildSwift/CMakeLists.txt]: Yeah, maybe I can rename the target name to `install-swift-toolchain-libllbuildSwift`.

Comment (ddunbar) [reply products/llbuildSwift/CMakeLists.txt]: meh, I'd rather keep it as is and make the path configurable later.

**Comment by aciidb0mb3r** [comment]
@swift-ci please smoke test

**Comment by aciidb0mb3r** [comment]
@swift-ci please smoke test

**Pull request merged**